.rubocop_todo.yml

```diff
@@ -1,32 +1,57 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2021-04-14 06:51:37 UTC using RuboCop version 0.90.0.
+# on 2021-08-16 15:31:43 UTC using RuboCop version 1.19.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 26
-# Configuration parameters: IgnoredMethods.
+# Offense count: 7
+# Configuration parameters: AllowedMethods.
+# AllowedMethods: enums
+Lint/ConstantDefinitionInBlock:
+  Exclude:
+    - 'spec/faraday/composite_read_io_spec.rb'
+    - 'spec/faraday/options/options_spec.rb'
+    - 'spec/faraday/rack_builder_spec.rb'
+    - 'spec/faraday/request/instrumentation_spec.rb'
+
+# Offense count: 11
+# Configuration parameters: AllowComments, AllowEmptyLambdas.
+Lint/EmptyBlock:
+  Exclude:
+    - 'spec/faraday/connection_spec.rb'
+    - 'spec/faraday/rack_builder_spec.rb'
+    - 'spec/faraday/response_spec.rb'
+
+# Offense count: 16
+# Configuration parameters: IgnoredMethods, CountRepeatedAttributes.
 Metrics/AbcSize:
   Max: 42
 
-# Offense count: 5
+# Offense count: 3
 # Configuration parameters: CountComments, CountAsOne.
 Metrics/ClassLength:
-  Max: 256
+  Max: 226
 
-# Offense count: 15
+# Offense count: 12
 # Configuration parameters: IgnoredMethods.
 Metrics/CyclomaticComplexity:
   Max: 13
 
-# Offense count: 43
-# Configuration parameters: CountComments, CountAsOne, ExcludedMethods.
+# Offense count: 27
+# Configuration parameters: CountComments, CountAsOne, ExcludedMethods, IgnoredMethods.
 Metrics/MethodLength:
-  Max: 37
+  Max: 33
 
-# Offense count: 9
+# Offense count: 8
 # Configuration parameters: IgnoredMethods.
 Metrics/PerceivedComplexity:
   Max: 14
+
+# Offense count: 3
+Style/DocumentDynamicEvalDefinition:
```

Comment:
I want to talk a bit about this rule, since the fix will have visual impact, and we'd like to pick the nicest alternative.